.travis.yml

```diff
@@ -2,7 +2,7 @@ language: node_js
 before_install:
   - npm install -g npm@latest
 node_js:
-  - 4
```

Comment:
Yeah, I think as long as it's LTS we should support it – there are a surprising amount of people is still on Node 4.